karton_whatvpn/karton_whatvpn.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+
+from artemis import utils, load_risk_class
+from artemis.binds import TaskStatus, TaskType, Service
+from artemis.module_base import ArtemisBase
+from artemis.task_utils import get_target_url, get_target_host
+from karton.core import Task
+import string
+
+from extra_modules_config import ExtraModulesConfig
+
+logger = utils.build_logger(__name__)
+
+@load_risk_class.load_risk_class(load_risk_class.LoadRiskClass.LOW)
+class WhatVPN(ArtemisBase):
+    """ 
+    Runs what-vpn -> SSL VPN identifier
+    """
+    
+    identity = "what-vpn"
+    filters = [
+        {"type": TaskType.IP.value}
+    ]
+
+    def _process(self, current_task: Task, host: str) -> None:
+        output = subprocess.run(
+            [
+                "what-vpn",
+                "--keep-going-after-exception",
+                "--timeout",
+                ExtraModulesConfig.WHATVPN_TIMEOUT_S,
+                host
+            ],
+            capture_output=True
+        )
+        output = output.stdout.decode("utf-8")
+        detected_vpn = []
+
+        error_messages = ["error", "timeout", "no match"]
+        if any(msg in output for msg in error_messages):
+            status = TaskStatus.OK
```

Comment:
TaskStatus.ERROR

Reply:
changed the logic to return separate OK and ERROR statuses